gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannelTest.java

```diff
@@ -1195,6 +1051,16 @@ public void testReadCachedFooterPartiallyWithInplaceSeek() throws Exception {
     buffer.clear();
     readChannel.position(footerOffset);
     readChannel.read(buffer);
+    verify(fakeService, times(1))
+        .readObject(
```

Comment:
Please change the comment to ensure that the additional calls to GCS is intentional

Reply:
Fixed